src/main/java/org/team1540/robot2024/subsystems/indexer/Indexer.java

```diff
@@ -0,0 +1,60 @@
+package org.team1540.robot2024.subsystems.indexer;
+
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.littletonrobotics.junction.Logger;
+import org.team1540.robot2024.util.LoggedTunableNumber;
+import static org.team1540.robot2024.Constants.Indexer.*;
+
+
+public class Indexer extends SubsystemBase {
+    private final IndexerIO indexerIO;
+    private final IndexerIOInputsAutoLogged indexerInputs = new IndexerIOInputsAutoLogged();
+    private final boolean TUNING = true;
```

Comment:
There's already a constant called `tuningMode` in the constants file, use that instead of this